client/modules/IDE/components/Toolbar.test.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import lodash from 'lodash';
-
-
 import { ToolbarComponent } from './Toolbar';
+import { i18ntest } from '../../../i18n-test';
```

Comment:
If you're not using the `i18ntest` variable, then there's no need to assign it here. You can just do this:

`import '../../../i18n-test';`